tiberius/src/protocol/codec/token/token_row.rs

```diff
@@ -6,13 +6,38 @@ use crate::{
     },
 };
 use std::sync::Arc;
+use tokio::io::AsyncReadExt;
 
 #[derive(Debug)]
 pub struct TokenRow {
     pub meta: Arc<TokenColMetaData>,
     pub columns: Vec<ColumnData<'static>>,
 }
 
+struct RowBitmap {
+    data: Vec<u8>,
+}
+
+impl RowBitmap {
+    fn is_null(&self, i: usize) -> bool {
+        let index = i / 8;
+        let bit = i % 8;
+
+        self.data[index] & (1 << bit) > 0
+    }
```

Comment:
interesting, so the null columns have a `1` bit`, instead of the reverse, right?